README.md

```diff
@@ -36,90 +40,51 @@ The Nulecule specification enables complex applications to be defined, packaged
-1. Download the application files using `atomic install`
-
-        [sudo] atomic install projectatomic/helloapache
-
-1. Rename `answers.conf.sample`
-
-        mv answers.conf.sample answers.conf
-
-1. Edit `answers.conf`, review files if desired and then run
-
-        $ [sudo] atomic run projectatomic/helloapache
-        ...
-        helloapache
-
-## Test
-Any of these approaches should create a kubernetes pod or a running docker container.
-
-With a kubernetes pod, once its state is "Running" curl the minion it's running on.
-
-```
-$ kubectl get pod helloapache
-POD                IP                  CONTAINER(S)       IMAGE(S)           HOST                LABELS              STATUS
-helloapache        172.17.0.8          helloapache        centos/httpd       10.3.9.216/         name=helloapache   Running
-$ curl 10.3.9.216
-<bunches_of_html_goodness>
-```
-
-If you test the docker provider, once the container is running, curl the port on your localhost.
+This example is a single container application based on the centos/httpd image, but you can use your own.
 
+A quick example of this being used are launching the `projectatomic/helloapache` example:
+
+```bash
+▶ sudo atomicapp run projectatomic/helloapache --destination helloapache
+INFO   :: Atomic App: 0.5.2 - Mode: Run
+INFO   :: Unpacking image projectatomic/helloapache to helloapache
+INFO   :: Skipping pulling docker image: projectatomic/helloapache
+INFO   :: Extracting Nulecule data from image projectatomic/helloapache to helloapache
+INFO   :: App exists locally and no update requested
+INFO   :: Using namespace default
+INFO   :: trying kubectl at /usr/bin/kubectl
+INFO   :: trying kubectl at /usr/local/bin/kubectl
+INFO   :: found kubectl at /usr/local/bin/kubectl
+INFO   :: Deploying to Kubernetes
+
+Your application resides in helloapache
+Please use this directory for managing your application
+
+▶ kubectl get po
+NAME                   READY     STATUS    RESTARTS   AGE
+helloapache            1/1       Running   0          6s
+k8s-etcd-127.0.0.1     1/1       Running   0          20d
+k8s-master-127.0.0.1   4/4       Running   0          23h
+k8s-proxy-127.0.0.1    1/1       Running   0          23h
 ```
-$ curl localhost
-<bunches_of_html_goodness>
-```
-
-Additional examples are available in the [examples](examples/) directory.
 
 ## Developer User Experience
 
-See the [Getting Started with Nulecule guide](docs/getting-started.md).
+See the [Getting Started with Nulecule guide](GETTING_STARTED.md).
 
 ## Implementations
 
-This is only a specification. Implementations may be written in any language. See [implementation guide](/docs/implementation_guide.md) for more details.
+This is only a specification. Implementations may be written in any language. See [implementation guide](IMPLEMENTATION_GUIDE.md) for more details.
 
-**Reference implementation** https://github.com/projectatomic/atomicapp
+**Reference implementation:** https://github.com/projectatomic/atomicapp
```

Comment:
Still would rather not use the words "reference implementation" here. Let's just list Atomic App as an implementation, but not the reference implementation.